app/src/main/java/com/example/farmeraid/data/QuotasRepository.kt

```diff
@@ -18,13 +20,18 @@ class QuotasRepository {
 
     data class Quota(
         val id: String,
-        val produceQuotaList : List<ProduceQuota>,
+        val produceQuotaList: List<ProduceQuota>,
+        val saleAmount: MutableMap<String, Int>
```

Comment:
Basically I mean to remove this saleAmount map, and instead just add a produceSaleAmount variable in ProduceQuota which is just of type Int.